lib/src/widgets/switch/ds_switch.widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// Animation duration for [DSSwitch].
+const Duration _animationDuration = Duration(milliseconds: 400);
```

Comment:
any reason to declare this outside of class?

Reply:
Done